postman/Get Broadcast Details.postman_collection.json

```diff
@@ -134,30 +127,26 @@
 						"broadcasts",
 						"total"
 					],
-					"query": [
-						{
-							"key": "status",
-							"value": "draft"
-						}
-					]
+					"query": [{
+						"key": "status",
+						"value": "draft"
+					}]
 				}
 			},
 			"response": []
 		},
 		{
 			"name": "Get Broadcast Totals  - Scheduled",
-			"event": [
-				{
-					"listen": "test",
-					"script": {
-						"id": "aebe59b2-ac5a-41af-9f56-21bd56794e16",
-						"exec": [
-							""
-						],
-						"type": "text/javascript"
-					}
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "aebe59b2-ac5a-41af-9f56-21bd56794e16",
+					"exec": [
+						""
+					],
+					"type": "text/javascript"
 				}
-			],
+			}],
```

Comment:
this test suite is no longer working.
<img width="1136" alt="screen shot 2018-11-30 at 10 09 22 am" src="https://user-images.githubusercontent.com/3092064/49297138-0a47b000-f488-11e8-92fd-dbccbc7949d4.png">

Reply:
@amberheilman are you sure you have drafts on that list?